src/mplfinance/_utils.py

```diff
@@ -262,6 +302,109 @@ def _construct_candlestick_collections(dates, opens, highs, lows, closes, market
 
     return rangeCollection, barCollection
 
+def _construct_renko_collections(dates, highs, lows, volumes, renko_params, closes, marketcolors=None):
+    """Represent the price change with bricks
+
+    Parameters
+    ----------
+    dates : sequence
+        sequence of dates
+    highs : sequence
+        sequence of high values
+    lows : sequence
+        sequence of low values
+    renko_params : dictionary
+        type : type of renko chart
+        brick_size : size of each brick
+        atr_legnth : length of time used for calculating atr
+    closes : sequence
+        sequence of closing values
+    marketcolors : dict of colors: up, down, edge, wick, alpha
+
+    Returns
+    -------
+    ret : tuple
+        rectCollection
+    """
+    if marketcolors is None:
+        marketcolors = _get_mpfstyle('classic')['marketcolors']
+        print('default market colors:',marketcolors)
+    
+    brick_size = renko_params['brick_size']
+    atr_length = renko_params['atr_length']
+
+    if atr_length > len(closes):
+        raise ValueError("Specified atr_length is larger than the length of the dataset: " + str(len(closes)))
+
+    if brick_size == 'atr':
+        brick_size = _calculate_atr(atr_length, highs, lows, closes)
+
+    alpha  = marketcolors['alpha']
+
+    uc     = mcolors.to_rgba(marketcolors['candle'][ 'up' ], 1.0)
+    dc     = mcolors.to_rgba(marketcolors['candle']['down'], 1.0)
+    euc     = mcolors.to_rgba(marketcolors['edge'][ 'up' ], 1.0)
+    edc     = mcolors.to_rgba(marketcolors['edge']['down'], 1.0)
+
+    cdiff = [(closes[i+1] - closes[i])/brick_size for i in range(len(closes)-1)] # fill cdiff with close price change
+
+    bricks = [] # holds bricks, 1 for down bricks, -1 for up bricks
+    new_dates = [] # holds the dates corresponding with the index
+    new_volumes = [] # holds the volumes corresponding with the index.  If more than one index for the same day then they all have the same volume.
+
+    prev_num = 0
+    start_price = closes[0]
+
+    volume_cache = 0 # holds the volumes for the dates that were skipped
+    
+
+    for i in range(len(cdiff)):
+        num_bricks = abs(int(round(cdiff[i], 0)))
+
+        if num_bricks != 0:
+            new_dates.extend([dates[i]]*num_bricks)
+            new_volumes.extend([volumes[i] + volume_cache]*num_bricks)
```

Comment:
`mpf.plot()` allows that volumes can be `None`, that is, the caller can pass in a dataframe that does not contain a 'Volumes' column, and that's fine as long as the caller does not also pass in `volume=True`.  

This code should handle the case of volumes being None.  Thank you.